CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Contributing to PollCord
+
+Thank you so much for taking an interest in contributing!
+
+Follow the set of guidelines below to contribute to PollCord!
+
+## Code of Conduct
+
+By participating, you are expected to uphold this code. Please report unacceptable behavior to guptabhishek785@gmail.com, ntgomes@gmail.com
+
+Prerequisites required before starting this project during the Fall of 2022.
```

Comment:
This section isn't really necessary, so I would remove this and the 3 numbered items below. Remember, the target audience for this repo is *anyone* who wants to contribute. They don't even have to be a part of NC State or do this for academic purposes. So having this doesn't really make any sense for contributors not affiliated with NC State.
If you had to include a prerequisite, just say that you need to read, understand, and abide by the repo's Code of Conduct, linking to the CODE_OF_CONDUCT.md